README.md

```diff
@@ -84,6 +90,35 @@ level logging.
 
 Note: Instead of using the `get` and `set` methods as seen above, you may also wish to use property getters and setters to make your code compatible with other libraries ie `request.session.data = request.body` and `const data = request.session.data` are also possible. However, if you want to have properties named `changed` or `deleted` in your session data, they can only be accessed via `session.get()` and `session.set()`. (Those are the names of internal properties used by the Session object)
 
+
+### Multiple sessions
+
+If you want to use multiple sessions, you have to supply an array of options when registering the plugin. It supports the same options as a single session but in this case, the `sessionKey` name is mandatory.
+
+```js
+fastify.register(require('@fastify/secure-session'), [{
+  sessionKey: 'mySession',
```

Comment:
Could you rename this to `name` or `sessionName`? Calling it `sessionKey` risk confusion with a cryptographic key.

Reply:
Good one 👍, pushed the changes.